packages/remix-dev/cli/run.ts

```diff
@@ -315,15 +313,79 @@ export async function run(argv: string[] = process.argv.slice(2)) {
           throw error;
         });
 
+      let installDeps = flags.install !== false && answers.install !== false;
+
       await commands.create({
         appTemplate: flags.template || answers.appTemplate,
         projectDir,
         remixVersion: flags.remixVersion,
-        installDeps: flags.install !== false && answers.install !== false,
-        useTypeScript:
-          flags.typescript !== false && answers.useTypeScript !== false,
+        installDeps,
+        useTypeScript: flags.typescript !== false,
         githubToken: process.env.GITHUB_TOKEN,
       });
+
+      let isTypeScript = fse.existsSync(path.join(projectDir, "tsconfig.json"));
+      let isDeno = fse.existsSync(path.join(projectDir, "deno.json"));
```

Comment:
`isDeno` won't reliably detect Deno since [Deno's configuration file can be named anything](https://deno.land/manual/getting_started/configuration_file).

`isTypescript` will be `false` for Deno projects, but I think that's ok for now. We just don't support TS->JS conversion for Deno.

Reply:
ok cool, I think I misunderstood the original message from Pedro, sorry! aside, they can still convert if they _really_ want to by using `--no-typescript`